blog/2024-09-10-pants-2-22/index.mdx

```diff
@@ -2,7 +2,7 @@
   authors: [benjy]
   tags: [announcement]
   # The release happened on this date, despite what the file name implies (which is when this post was authored)
-  date: 2024-10-07
+  date: 2024-09-10
```

Comment:
Correct the date of this blog post to reflect when v2.22 was actually released.

cc @benjyw on whether this change is fine

Reply:
Without this change, the v2.22 release post appears *after* the survey's blog post.